src/modules/core/components/MembersList/MembersListItem.tsx

```diff
@@ -110,21 +112,27 @@ const MembersListItem = <U extends AnyUser = AnyUser>(props: Props<U>) => {
             size="s"
             colonyAddress={colonyAddress}
             address={walletAddress}
-            user={user}
+            user={userProfile}
             showInfo={!onRowClick || showUserInfo}
             domainId={domainId}
             notSet={false}
           />
         </div>
         <div className={styles.usernameSection}>
-          {displayName && (
-            <span className={styles.displayName} title={displayName}>
-              {displayName}
+          {userProfile.profile.displayName && (
```

Comment:
Are we totally sure that userProfile won't be null?

Reply:
I think so...? But I'll check anyway :)

Reply:
Apparently, I was correct. `useUser` will always return a correct user object, with **at least** the wallet address populated.

Eg:
```js
{
  profile: {
    walletAddress: "0x0....."
  }
}
```

So that means we can always expect that `userProfile` is set, and we can always destructure from it, without either TS screaming at us or the React component throwing an error.

Here's two `userProfile`'s logged, one has a username registered, the otherone doesn't:
![Screenshot from 2020-11-29 13-43-56](https://user-images.githubusercontent.com/1193222/100540954-5f6e5600-3249-11eb-9943-76081a0970b4.png)